controller/src/main/java/io/pravega/controller/store/task/ZKTaskMetadataStore.java

```diff
@@ -79,22 +80,16 @@ Void transferLock(final Resource resource,
             Stat stat = new Stat();
             byte[] data = client.getData().storingStatIn(stat).forPath(getTaskPath(resource));
             LockData lockData = LockData.deserialize(data);
-            if (lockData.isOwnedBy(oldOwner, oldThreadId)) {
-                lockData = new LockData(owner, threadId, lockData.getTaskData());
+            log.info("Transfer lock with the lockData.getHostId {}, oldOwner {}, oldThreadId {}, " +
```

Comment:
will this log line after the ZK set data(87-88) will be useful in cases of debugging?

Reply:
It was added before so that during the lock transfer it would be easier to track in case of any error before updating the client